model/dependencies.go

```diff
@@ -115,7 +122,7 @@ func (di *dependencyIncluder) expandDependencies(pair TVPair, depends []TaskUnit
 					}
 					projectTask := di.Project.FindTaskForVariant(t.Name, v.Name)
 					if projectTask != nil {
-						if projectTask.SkipOnRequester(di.requester) {
+						if utility.FromBoolPtr(projectTask.Disable) || projectTask.SkipOnRequester(di.requester) {
```

Comment:
Can we make a function for projectTasks IsDisabled?

Reply:
done